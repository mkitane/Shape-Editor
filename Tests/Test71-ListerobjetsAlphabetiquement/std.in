C r1 1 1 1
C c1 2 2 2
LIST
EXIT