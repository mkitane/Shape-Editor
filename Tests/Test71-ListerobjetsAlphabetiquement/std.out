OK
OK
C c1 2 2 2
C r1 1 1 1
